finality-aleph/src/network/base_protocol/handler.rs

```diff
@@ -40,6 +51,21 @@ impl PeerInfo {
     }
 }
 
+impl<B> From<PeerInfo> for ExtendedPeerInfo<B>
+where
+    B: Block<Hash = BlockHash>,
+    B::Header: Header<Number = BlockNumber>,
+{
+    fn from(peer_info: PeerInfo) -> Self {
+        ExtendedPeerInfo {
+            roles: peer_info.role.into(),
+            // this is never actually inspected, so return dummy values
+            best_hash: H256([0; 32]),
+            best_number: 0,
```

Comment:
use const with names `DUMMY_*`

Reply:
Actually, how hard would it be to get those values

Reply:
Added `const`s.

Annoyingly hard. It would require:
1. Connecting the base protocol with sync. They are created quite far apart, so that complexity would spread all over the code.
2. Making sync actually track this – we have that info there, but only implicitly at this point.
Imo completely not worth it, if we wanted to return similar info it would be better to create our own RPC. Also, as mentioned in the comment, this _never_ actually gets inspected, the only call checks only the length of the returned vector, even having vaguely correct `Roles` is a bit of overkill.